content/_components/onedrive/technical-notes.md

```diff
@@ -5,15 +5,25 @@ description: Technical Notes for the Microsoft OneDrive component
 icon: onedrive.png
 icontext: Microsoft OneDrive component
 category: onedrive
-ComponentVersion: 1.0.6
-updatedDate: 2022-09-23
+ComponentVersion: 2.0.0
+updatedDate: 2023-05-19
 redirect_from:
   - /components/onedrive/completeness-matrix.html
   - /components/onedrive/changelog.html
 ---
 
 ## Changelog
 
+### 2.0.0 (May 19, 2023)
+
+* New version of component, no backward compatibility! Written from scratch using TS and last versions of libraries
```

Comment:
* New version of the component, no backward compatibility! Written from scratch using TS and the last versions of libraries